### PR TITLE
README update - Getk12 patch 1 

### PR DESCRIPTION
**Pull request by getk12**
Updated README to reflect addition of new issue/feature templates, and some additional items.
Please review and share feedback.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,10 +38,10 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
...
 
-## Trademarks
+## Trademarks ##
 
 This project may contain trademarks or logos for projects, products, or services. Authorized use of Microsoft 
 trademarks or logos is subject to and must follow 
 [Microsoft's Trademark & Brand Guidelines](https://www.microsoft.com/en-us/legal/intellectualproperty/trademarks/usage/general).
 Use of Microsoft trademarks or logos in modified versions of this project must not cause confusion or imply Microsoft sponsorship.
-Any use of third-party trademarks or logos are subject to those third-party's policies.
+Any use of third-party trademarks or logos are subject to those third-party's policies.
```
Comment (daviddavis) [inline README.md:47]: nitpick - extraneous space at the end here.

**Comment by daviddavis** [comment]
This looks good to me overall. I am not totally sure about our sub-repos though. Maybe @mbearup you can review that section?

File: README.md [hunk 2]
```diff
@@ -1,7 +1,30 @@
 # Microsoft Linux Package Repositories
-This repository is intended to communicate status of [packages.microsoft.com](https://packages.microsoft.com) and to receive feedback on the service. Issues can be filed in the [issues](https://github.com/microsoft/linux-package-reposotories/issues) page. For security issues, refer to [SECURITY.md](SECURITY.md).
 
-## Contributing
+## Overview 
+
+The Microsoft Linux Package Repositories are hosted on PMC ([packages.microsoft.com](https://packages.microsoft.com)) service. The PMC service is intended to support package hosting for customers with clients running a distribution of the Linux OS. Microsoft builds and supports a variety of software products for Linux systems and makes them available via standard APT and YUM package repositories.
```
Comment (mbearup) [inline README.md:5]: recommend "distribution of Linux."
Avoids argument of whether Linux is OS vs Kernel.

File: README.md [hunk 3]
```diff
@@ -1,7 +1,30 @@
...
+
+The Microsoft Linux Package Repositories are hosted on PMC ([packages.microsoft.com](https://packages.microsoft.com)) service. The PMC service is intended to support package hosting for customers with clients running a distribution of the Linux OS. Microsoft builds and supports a variety of software products for Linux systems and makes them available via standard APT and YUM package repositories.  
+
+## Configuring the repository on your Linux system 
+
+See how to [host/install/upgrade](https://docs.microsoft.com/en-us/windows-server/administration/linux-package-repository-for-microsoft-software) Microsoft's Linux software using your distribution's standard package management tools.  
+
+Microsoft's Linux Software Repository is comprised of multiple sub-repositories: 
+
+**prod** – The Production sub-repository is designated for packages intended to be used in production. These packages are commercially supported by Microsoft under the terms of the applicable support agreement or program that you have with Microsoft.
```
Comment (mbearup) [inline README.md:13]: I would use plural here; there are multiple "prod" repositories (ubuntu, fedora, SUSE, etc). Perhaps clarify how the prod repositories are found (via top level folders: ubuntu/20.04/prod, or via their names: /repos/microsoft-ubuntu-focal-prod).

**Review by mbearup — CHANGES_REQUESTED** [review]
Recommend some changes to wording, particularly around "sub-repositories"

**Review by mbearup — APPROVED** [review]

**Pull request merged**